sandy/core/records.py

```diff
@@ -327,9 +327,40 @@ def write_int(x):
 
 
 def line_numbers(length, istart=1):
+    """
+    Line number creator
+
+    Parameters
+    ----------
+    length : 'int'
+        Number of lines.
+    istart : 'int', optional
+        Number from where start counting. The default is 1.
+
+    Returns
+    -------
+    ilines : `list`
+        List containing the number of each line.
+
+    Examples
+    --------
+    >>> np.array(line_numbers(1.0e6)).max()
+    99999
+    >>> np.array(line_numbers(1.0e6+1)).min()
+    1
+    >>> np.array(line_numbers(1.0e4+1)).max()
+    10001.0
```

Comment:
I don't like having float numbers while all outputs should be integers

Reply:
IT is because this specific part is a numpy array. The `(line_numbers(1.0e4+1)` is `int`